main.tf

```diff
@@ -52,6 +57,11 @@ module "obs_testnet_vpc" {
   create_iam_instance_profile_ssm_policy = "true"
   deploy_ec2_instance_db                 = false
   deploy_rds_db                          = true
+  xchain_settings = {
+    enabled      = false
+    docker_image = "omniops/xchain-indexer:latest"
+    config       = "staging"
```

Comment:
why is config staging here?

Reply:
we don't have a testnet config yet, and this is disabled now anyway we will have to update it with he proper values when we deploy to testnet